app/change_sets/hyrax/batch_edit_change_set.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Hyrax
+  class BatchEditChangeSet < WorkChangeSet
+    property :creator, virtual: true, multiple: true, required: false
```

Comment:
@mbklein this is somewhat parallel to your dynamic generation to about having to list these out, thoughts?

Reply:
These can not be dynamic because you don't know which type of objects are in the batch.

Reply:
Furthermore, the Batch edit page only allows a limited subset of fields to be edited.